packages/app/components/jsonschemaform/widgets/ConcatenatedWidget.tsx

```diff
@@ -82,7 +82,6 @@ export default function ConcatenatedWidget(props: WidgetProps) {
                     // update the concatenated value anytime this field blurs or changes
                     el.onblur = () => updateConcatenatedFieldValueFromFields(fields)
                     el.onchange = () => updateConcatenatedFieldValueFromFields(fields)
-                    el.onkeyup = () => updateConcatenatedFieldValueFromFields(fields)
```

Comment:
This was causing a small, but perceptible lag when dealing with concatenated fields. I think updating the concatenated field on blur and change is good enough